common/api/summary/ui-components.exports.csv

```diff
@@ -108,13 +108,16 @@ internal;ContextMenuItem
 internal;ContextMenuProps 
 beta;ControlledSelectableContent(props: ControlledSelectableContentProps): JSX.Element
 beta;ControlledSelectableContentProps
-beta;ControlledTree(props: ControlledTreeProps): JSX.Element
-beta;ControlledTreeProps 
+public;ControlledTree(props: ControlledTreeProps): JSX.Element
+public;ControlledTreeProps 
 public;ConvertedPrimitives
 internal;convertPrimitiveRecordToString(record: PropertyRecord): string | Promise
 internal;CubeHover
-beta;CubeNavigationAid 
-beta;CubeNavigationAidProps 
+public;CubeNavigationAid 
```

Comment:
There are other properties that start with "NavCube" instead of "CubeNavigation" I kind of like the shorter version.

Reply:
I'd rather keep it consistent with the component  name.